tests/lora/utils.py

```diff
@@ -65,6 +66,12 @@ def check_if_lora_correctly_set(model) -> bool:
     return False
 
 
+def populate_meta_state_dict_with_dummy(state_dict):
```

Comment:
Is the purpose of this function is to move the state dict out from meta device? name is misleading if it is the case

Reply:
That is right!

I changed to `initialize_dummy_state_dict()` in [ca5a1d5](https://github.com/huggingface/diffusers/pull/9510/commits/ca5a1d5194058f92c6b7524dcb4d2d49f6cc7885).

Does that work for you?